src/etc/rc.bootup

```diff
@@ -227,6 +229,9 @@ if (!$debugging) {
 /* re-make hosts file after configuring interfaces */
 system_hosts_generate();
 
+/* add configured trusted CA certificates to /etc/ssl/certs */
+ca_trustedcerts_generate();
+
```

Comment:
It's safer to do it at boot time to ensure it's consistent. It's not like it's a huge amount of work or writes.